app/views/admin/action_pages/_panel_content.html.erb

```diff
@@ -31,24 +31,24 @@
           <ul class="small">
             <li>
               <strong>Want awesome section headers?</strong>
-                      Use this Markdown snippet to make some awesome section headers: <code>### **Amaze**</code>
-                    </li>
-                    <li>
-                      <strong>Using images in your description?</strong>
-                      Make them extra-wide using <a href="#" id="protip2" data-toggle="popover"  data-trigger="click" data-html="false" data-content='<img class="full-width" src="http://image-url" />'>this syntax</a>.
-                    </li>
-                    <li>
-                      <strong>Want a pretty letter?</strong>
-                      Use <a href="#" id="protip3" data-toggle="popover"  data-trigger="click" data-html="true" data-content='<pre class="small">``` letter<br>Type your letter text here!<br>```</pre>'>this syntax</a> and your text will appear formatted nicely! Note: when you use this the petition letter will magically disappear.
-                    </li>
-                  </ul>
-                  <p class="small">
-                  </p>
-                </div>
-              </div>
+              Use this Markdown snippet to make some awesome section headers: <code>### **Amaze**</code>
+            </li>
+            <li>
+              <strong>Using images in your description?</strong>
+              Make them extra-wide using <a href="#" id="protip2" data-toggle="popover"  data-trigger="click" data-html="false" data-content='<img class="full-width" src="http://image-url" />'>this syntax</a>.
+            </li>
+            <li>
+              <strong>Want a pretty letter?</strong>
+              Use <a href="#" id="protip3" data-toggle="popover"  data-trigger="click" data-html="true" data-content='<pre class="small">``` letter<br>Type your letter text here!<br>```</pre>'>this syntax</a> and your text will appear formatted nicely! Note: when you use this the petition letter will magically disappear.
+            </li>
+          </ul>
+          <p class="small">
+          </p>
```

Comment:
might as well just keep this opening and closing of the `p` tag on the same line